README.md

```diff
@@ -298,7 +303,7 @@ the function or class docstring to get more info or go straight to our [API docs
 
 ## Issuing
 
-### Query IssuingBins
+### Query IssuingProducts
 
 To take a look at the sub-issuer BINs available to you, just run the following:
```

Comment:
`To take a look at the sub-issuer card products available to you, just run the following:`

Da um search all em `bin` para poder alterar para `product`, ainda existem muitos lugares com `bin`.